components/resources/ai_chat_ui_strings.grdp

```diff
@@ -55,13 +58,13 @@
      Retry
   </message>
   <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_DEFAULT" desc="AI Chat intro message for the default model">
-    I'm here to help. What can I assist you with today?
+    Hi, I'm Leo. I'm a fully hosted AI assistant by Brave. I'm  powered by Llama 13B, a model created by Meta to be performant and applicable to many use cases.
   </message>
   <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_LEO_EXPANDED" desc="AI Chat intro message for the expanded model">
-    I have a vast base of knowledge and a large memory able to help with more complex challenges.
+    Hi, I'm Leo. I'm a fully hosted AI assistant by Brave.  I'm powered by Llama 70B, a model created by Meta to handle more advanced tasks than 13B.
```

Comment:
Super nit: there's an extra space between the sentences.

Reply:
that's not good, will fix. Thanks for spotting that!